src/Metadata/Property/Factory/SerializerPropertyMetadataFactory.php

```diff
@@ -203,6 +204,18 @@ private function getClassSerializerGroups(string $class): array
             $groups[] = $serializerAttributeMetadata->getGroups();
         }
 
+        if (true === $this->normalizeChildClassAttributeGroups) {
+            foreach ($serializerClassMetadata->getReflectionClass()->getAttributes() as $reflectionAttribute) {
+                if (DiscriminatorMap::class !== $reflectionAttribute->getName()) {
```

Comment:
is discriminatorMap an attribute of the serializer? not sure I understand how this works.